src/main/java/uk/gov/hmcts/reform/professionalapi/controller/constants/ProfessionalApiConstants.java

```diff
@@ -10,7 +10,7 @@ private ProfessionalApiConstants() {
 
     public static final String EMAILREGEX = "^[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+(?:\\.[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+)"
             + "*@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]+)*$";
```

Comment:
why we are maintaining two email regex ..can you please try to keep only one email regex across the flow?

Reply:
This has been addressed here: https://github.com/hmcts/rd-professional-api/pull/829/commits/ae07de40cb925d0ab1544718f3533808214456c8